src/main/java/teammates/ui/controller/InstructorCourseEnrollSaveAction.java

```diff
@@ -43,7 +44,7 @@ public ActionResult execute() throws EntityDoesNotExistException {
                                                                     courseId, students, hasSection, studentsInfo);
             
             statusToAdmin = "Students Enrolled in Course <span class=\"bold\">[" 
-                            + courseId + "]:</span><br>" + (studentsInfo).replace("\n", "<br>");
+                            + courseId + "]:</span><br>" + Sanitizer.sanitizeForHtml((studentsInfo)).replace("\n", "<br>");
```

Comment:
Thanks for pointing that out. Resolved in 9902d2cd5e31056f8db8fae892b3bc097b866789.